model/RepositoryService.php

```diff
@@ -178,9 +180,22 @@ public function restore(Revision $revision)
             $this->getModel()->getRdfInterface()->add($triple);
         }
 
+        $item = $this->getQtiService()->getDataItemByRdfItem($resource);
+        $this->getUpdatedItemEventDispatcher()->dispatch($item, $resource);
```

Comment:
I think this a bad idea to dispatch `itemUpdatedEvent` at restore. Because you cannot planned all listeners.

Reply:
> you cannot planned all listeners

What is that mean?